docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py

```diff
@@ -18,7 +21,7 @@
 class Author(UUIDBase):
     name: Mapped[str]
     dob: Mapped[date]
-    books: Mapped[list["Book"]] = relationship(back_populates="author", lazy="selectin")
+    books: Mapped[list[Book]] = relationship(back_populates="author", lazy="selectin")
```

Comment:
To run on 3.8 this would have to be:

```py
from typing import List
...
class Author(UUIDBase):
    name: Mapped[str]
    dob: Mapped[date]
    books: Mapped[List[Book]] = relationship(back_populates="author", lazy="selectin")
```

Reply:
I seem to be running in the following ruff error:

`docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py:24:19: UP006 Use `list` instead of `List` for type annotation`

I am not sure why other scripts, which also use `List` do not trigger the same ruff errors.  See e.g. `/docs/examples/data_transfer_objects/factory/included_fields.py` 

Any tips?

Reply:
Yeah, we disable that lint for the file in pyproject:
https://github.com/litestar-org/litestar/blob/d75efa7e79be5a252920aa29f204eb0b310d62a4/pyproject.toml#L397

Reply:
Ok thanks for the info